js/dcf-modal.js

```diff
@@ -129,7 +129,18 @@ export class DCFModal {
 
     // Trigger pre-open modal event for this modal to allow event listeners to handle
     const preOpenEventName = `ModalPreOpenEvent_${modalId}`;
-    document.dispatchEvent(new CustomEvent(preOpenEventName));
+    // Getting the button from dom which button got clicked
+    const openButtonElement = document.getElementById(openBtnId);
+    let preOpenEvent = new CustomEvent(preOpenEventName);
+    // If there is a button then it's ID gets passed as a arguemnt in the CustomEvent
+    if(openButtonElement){
```

Comment:
Can you add spaces on this line to make it formatted like `if (openButtonElement) {` and can you add `openButtonElement === null` just so we know what we are checking for.